### PR TITLE
Fixed Taiko flashlight, flashlight now appears before the start of the map, allowing objects to be seen more closely to stable.

### PR DESCRIPTION
**Pull request by IzTheCodeMan**
A fix for [https://github.com/ppy/osu/issues/20905](url)

### CODE REVIEW

**Comment by IzTheCodeMan** [comment]
https://youtu.be/lEKqPo5lZR4 Video of changes working.

**Comment by IzTheCodeMan** [comment]
Sorry about it looking like spam, I made some mistakes with my fork and local branch, tis fixed now!

File: osu.Game.Rulesets.Taiko/Mods/TaikoModFlashlight.cs [hunk 1]
```diff
@@ -79,7 +79,11 @@ protected override void Update()
 
                     flashlightProperties.Validate();
                 }
+                else
+                {
+                    UpdateFlashlightSize(GetSize());
```
Comment (bdach) [inline osu.Game.Rulesets.Taiko/Mods/TaikoModFlashlight.cs:84]: At least this is in the correct neighbourhood of a proper fix now, but it's still wrong. It makes `flashlightProperties` entirely useless, nullifying the optimisations from the `LayoutValue`.

The actual problem here is that this logic regressed in a recent pull. I'll describe in more detail in the issue thread.

**Comment by bdach** [comment]
Closing this, as the change is wrong and the history is a horrible trash fire (it should not take 15 commits to make this change). I also don't understand why the commits here are apparently authored by @WyvernX-YT.

**Pull request closed without merging**